### PR TITLE
Hometask8: ReactJS stack useful libraries & packages

### PR DESCRIPTION
**Pull request by ketanthakkar**
Configure Airbnb Eslint.
Added flow instead of Prop Types in some components.
Added styled components.

Please review

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules/
+dist/
+webpack.*
+/server
```
Comment (MichalWierzba) [inline .eslintignore]: I'd suggest linting `webpack` files and `server` dir.

`coverage` and `lib` directories can be ignored.

File: src/index.test.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 test('', () => {
-  expect("");
-});
\ No newline at end of file
+  expect('');
+});
```
Comment (MichalWierzba) [inline src/index.test.js]: I think this file should be removed ;)

File: test/Content.test.js [hunk 3]
```diff
@@ -1,37 +1,34 @@
...
 describe('should render Content component', () => {
-    const tree = shallow(
-            <MovieContent movies={movies.data} />
-        );
-    test('Snapshot test', () => {   
-        expect(tree).toMatchSnapshot();
-    });
-
+  const tree = shallow(
+            <Content movies={movies.data} />,
```
Comment (MichalWierzba) [inline test/Content.test.js]: Looks like indentation issue.

File: test/actions.test.js [hunk 4]
```diff
@@ -1,111 +1,107 @@
...
-            moviesActions.requestMovies,
-            moviesActions.receiveMovies(response)
-        ];
-    
-        return store.dispatch(moviesActions.fetchMovies())
-          .then(() => {
-            expect(store.getActions()).toEqual(expected);
-          });
+  it('should call receiveMovies on success', () => {
+    fetchMock.getOnce('http://example.com/movies?search=fifty&searchBy=title&sortBy=release_date&sortOrder=desc', {
```
Comment (MichalWierzba) [inline test/actions.test.js]: Is `example.com` expected here?

**Review by MichalWierzba — APPROVED** [review]
👍

**Comment by ketanthakkar** [comment]
@thetreew Done changes as per your comment.

File: package.json [hunk 5]
```diff
@@ -12,7 +12,12 @@
     "test": "jest",
     "test:watch": "jest --watchAll",
     "test:coverage": "npm run test -- --coverage",
-    "test:open": "cypress open"
+    "test:open": "cypress open",
+    "lint:fix": "eslint ./ --cache --ignore-pattern .gitignore --fix",
+    "lint": "eslint ./ --cache --ignore-pattern .gitignore",
+    "build": "babel src/ -d lib/",
```
Comment (MichalWierzba) [inline package.json]: What is the purpose of this script?

File: .eslintignore (hunk 1 above)
Comment (MichalWierzba) [reply .eslintignore]: I meant that `coverage` and `lib` dirs should be added to this file ;)

**Comment by MichalWierzba** [comment]
@ketanthakkar Looks good to me 👍

File: package.json (hunk 5 above)
Comment (ketanthakkar) [reply package.json]: Sorry by mistake added but, forget to remove :)

**Comment by ketanthakkar** [comment]
@thetreew Done changes. :)

**Comment by MichalWierzba** [comment]
@ketanthakkar Do you plan add `reselect` library?

**Comment by ketanthakkar** [comment]
@thetreew Added `reselect`

**Comment by MichalWierzba** [comment]
Selectors look good :+1:


I'd suggest adding tests for selectors.

**Comment by ketanthakkar** [comment]
@thetreew Done changes and added tests for selectors

**Comment by MichalWierzba** [comment]
@ketanthakkar Thanks for introducing all the changes.

Please merge :)

**Pull request merged**